hydromt_fiat/fiat.py

```diff
@@ -221,6 +221,7 @@ def setup_vulnerability(
                 continent is not None
             ), "Please specify the continent when using the JRC vulnerability curves."
             self.vf_ids_and_linking_df["continent"] = continent.lower()
+            unit = "meters"
```

Comment:
If you're defining a default here, it's nicer to also use Units.meters.value here then instead of hardcoding "meters"